webapp/channels/src/components/channel_view/channel_view.tsx

```diff
@@ -117,9 +116,12 @@ export default class ChannelView extends React.PureComponent<Props, State> {
                     <div
                         className='channel-archived__message'
                     >
-                        <FormattedMarkdownMessage
-                            id='create_post.deactivated'
-                            defaultMessage='You are viewing an archived channel with a **deactivated user**. New messages cannot be posted.'
+                        <FormattedMessage
+                            id='createPost.deactivated'
```

Comment:
Quick curiosity question: is the change from `create_post.deactivated` to `createPost.deactivated` prompted due to error?

Same goes to `archivedChannelMessage` to `archivedChannel_Message`.

Not wrong, but curious if the id name changes were needed.

Reply:
Its was a recommendation because mmjstool which we use to extract translations doesnt pick up changes in messages, so a recommended for ease with contributors to just change the id which the tool will pick up